dist/s_function

```diff
@@ -38,7 +38,18 @@ for f in `find bench examples ext src test -name '*.c' -o -name '*._inline.h'`;
 	file_parse $f | sed "s=^=$f:="
 done | python dist/s_function_loop.py |
     egrep '\{@[^@]*(WT_RET[_A-Z]*)\([^@]*(WT_ERR[_A-Z]*)\(.*err:' |
-    sed -e 's/^\([^:]*\): *\([^:]*\):.*/\1:\2: mix of returns and jump to the error label within a loop/'
+    sed -e 's/^\([^:]*\): *\([^:]*\):.*/\1:\2: mix of returns and jump to the error label within a loop/' ||
+	exit 1
```

Comment:
I'd get rid of this (`|| exit 1`) part of the change, there's no reason to think this particular `sed` command will fail.

Reply:
ahh yep! sorry was trying to accommodate for the lack of check on `dist/s_function_loop.py`, will remove it thanks!